addons/mail/static/src/core/common/message_model.js

```diff
@@ -12,8 +12,13 @@ import { _t } from "@web/core/l10n/translation";
 import { user } from "@web/core/user";
 import { omit } from "@web/core/utils/objects";
 import { url } from "@web/core/utils/urls";
+import { markdown } from "@mail/utils/common/markdown";
+import { markup } from "@odoo/owl";
 
 const { DateTime } = luxon;
+
+const parser = new DOMParser();
```

Comment:
What's the difference between instantiating a new one for each usage and having a global one?

Reply:
Instantiate it once instead of two? I guess it's not that much of an optimization.
Maybe it could be a static of message class?